ts/features/whatsnew/screen/FastLoginWhatsNew.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from "react";
+import I18n from "i18n-js";
+
+import { Body } from "../../../components/core/typography/Body";
+import { VSpacer } from "../../../components/core/spacer/Spacer";
+import { H3 } from "../../../components/core/typography/H3";
+import { Label } from "../../../components/core/typography/Label";
+import { Link } from "../../../components/core/typography/Link";
+import { openWebUrl } from "../../../utils/url";
+export const FastLoginWhatsNewBody = (
+  <>
+    <Body>{I18n.t("fastLogin.whatsNew.subtitle")}</Body>
+    <VSpacer size={24} />
```

Comment:
It'd be better to create a style for spacer size, padding and so on. This way you have a single source of truth for this and a single point where to change values, just in case.